### ISSUE TITLE
Tinderauth failed: Server returned status code 404

### ISSUE THREAD

**Issue by alexmoreno** [tinderjs/tinderauth#7]
``` Tinderauth failed: Server returned status code 404 from https://www.facebook.com/intern/common/referer_frame.php ```

My email and password are correct. Why it's happening?

**Comment by thiaguerd** [comment]
i have same problem here

**Comment by baskervald** [comment]
Same here, which is a shame because I needed to update my bio and have no access to my phone right now (and tinder-desktop doesn't seem to have an option for it).

I might just have to fork tinder-desktop.

**Comment by mayeaux** [comment]
Tinder made an update, you guys may have to fork this and fix it. You can check tinder-desktop for the latest version of how to get a token.

**Comment by AndrewPrifer** [comment]
Hey guys, here's a short and sweet (~30 lines) working implementation in Python: https://git.io/vMNgR. I guess you should be able to translate the Robobrowser commands into Zombie.

**Comment by AndrewPrifer** [comment]
TLDR; The issue is probably with Zombie.
So I looked into the code and tried to fix things. The "Server returned status code 404" issue is easily fixed by using the right user agent (Mozilla/5.0 (Linux; U; en-gb; KFTHWI Build/JDQ39) AppleWebKit/535.19 (KHTML, like Gecko) Silk/3.16 Safari/535.19). After fixing the form inputs (the fields should be looked up based on name and not id, those will vary), I found a strange issue that could be a bug in Zombie. If you leave runScripts on, the whole response body is overwritten with gibberish (probably the source code of some script on the page) and the __CONFIRM__ button won't be found. But you need to run JavaScript on the page to get the XHR working after the button is pressed to get the access token. Also, if you turn on JavaScript after the page is loaded so that it doesn't get overwritten but you can still press the button, you get this strange error: "Tinderauth failed: Invalid protocol: fb464891386855067:".
